examples/contrib/cifar10/main.py

```diff
@@ -324,31 +370,19 @@ def train_step(engine, batch):
     trainer = Engine(train_step)
     trainer.logger = logger
 
-    to_save = {"trainer": trainer, "model": model, "optimizer": optimizer, "lr_scheduler": lr_scheduler}
     metric_names = [
         "batch loss",
     ]
 
     common.setup_common_training_handlers(
         trainer=trainer,
         train_sampler=train_sampler,
-        to_save=to_save,
-        save_every_iters=config["checkpoint_every"],
-        save_handler=get_save_handler(config),
         lr_scheduler=lr_scheduler,
         output_names=metric_names if config["log_every_iters"] > 0 else None,
         with_pbars=False,
         clear_cuda_cache=False,
     )
 
-    resume_from = config["resume_from"]
-    if resume_from is not None:
-        checkpoint_fp = Path(resume_from)
-        assert checkpoint_fp.exists(), f"Checkpoint '{checkpoint_fp.as_posix()}' is not found"
-        logger.info(f"Resume from a checkpoint: {checkpoint_fp.as_posix()}")
-        checkpoint = torch.load(checkpoint_fp.as_posix(), map_location="cpu")
-        Checkpoint.load_objects(to_load=to_save, checkpoint=checkpoint)
-
```

Comment:
I do not expect to see these changes. Why should we move this code elsewhere ?

Reply:
`to_save` needs to be updated as we will update `lr_scheduler` if `use_lr_finder` is true